SierraBib.rb

```diff
@@ -427,34 +464,10 @@ def lang008
     return [code, language]
   end
 
-
-  def oclcnum
```

Comment:
I took this out because adding :oclcnum to the class attr_accessor made it redundant. 

Highlighting in case you were doing it here for a reason (I haven't really been thinking about class vs instance attributes/methods and private methods, etc.)

Reply:
I agree with all the above and have made the suggested changes.